mypy/build.py

```diff
@@ -1012,6 +1038,12 @@ class State:
     # If caller_state is set, the line number in the caller where the import occurred
     caller_line = 0
 
+    # If True, indicate that the public interface of this module is unchanged
+    externally_same = True
+
+    # Contains a hash of the public interface in incremental mode
+    interface_hash = "never_set"  # type: str
```

Comment:
Oh, whoops -- I added that in to help with debugging, and forgot to revert that to an empty string. I'll make sure to do that.